game/scripts/vscripts/Sections/Chapter4/SectionOutpost.ts

```diff
@@ -99,27 +113,36 @@ function onStart(complete: () => void) {
             }),
             tg.wait(3),
 
-            // Part 2: Take outpost
-            // TODO: Camera pan on outpost
-            tg.immediate(_ => {
-                goalTakeOutpost.start();
+            tg.audioDialog(LocalizationKey.Script_4_Outpost_5, LocalizationKey.Script_4_Outpost_5, ctx => ctx[CustomNpcKeys.SlacksMudGolem]),
 
-                const dmgToDestroyTower = CreateDamageInfo(playerHero, playerHero, playerHero.GetAbsOrigin(), playerHero.GetAbsOrigin(), 9999, 9999);
+            tg.withHighlights(tg.seq([
+                tg.panCameraLinear(_ => playerHero.GetAbsOrigin(), outpostLocation, 2),
+                tg.audioDialog(LocalizationKey.Script_4_Outpost_6, LocalizationKey.Script_4_Outpost_6, ctx => ctx[CustomNpcKeys.SunsFanMudGolem]),
+                tg.audioDialog(LocalizationKey.Script_4_Outpost_7, LocalizationKey.Script_4_Outpost_7, ctx => ctx[CustomNpcKeys.SlacksMudGolem]),
 
-                const direTopTower1 = Entities.FindByName(undefined, "dota_badguys_tower1_top") as CDOTA_BaseNPC_Building | undefined;
-                if (direTopTower1 && unitIsValidAndAlive(direTopTower1)) {
-                    direTopTower1.TakeDamage(dmgToDestroyTower);
-                }
+                // Part 2: Take outpost
+                tg.immediate(_ => {
+                    goalTakeOutpost.start();
 
-                const direTopTower2 = Entities.FindByName(undefined, "dota_badguys_tower2_top") as CDOTA_BaseNPC_Building | undefined;
-                if (direTopTower2 && unitIsValidAndAlive(direTopTower2)) {
-                    direTopTower2.TakeDamage(dmgToDestroyTower);
-                }
-            }),
+                    const dmgToDestroyTower = CreateDamageInfo(playerHero, playerHero, playerHero.GetAbsOrigin(), playerHero.GetAbsOrigin(), 9999, 9999);
 
-            tg.completeOnCheck(_ => {
-                return direOutpost.GetTeam() === DotaTeam.GOODGUYS;
-            }, 1),
+                    const direTopTower1 = Entities.FindByName(undefined, "dota_badguys_tower1_top") as CDOTA_BaseNPC_Building | undefined;
+                    if (direTopTower1 && unitIsValidAndAlive(direTopTower1)) {
+                        direTopTower1.TakeDamage(dmgToDestroyTower);
+                    }
+
+                    const direTopTower2 = Entities.FindByName(undefined, "dota_badguys_tower2_top") as CDOTA_BaseNPC_Building | undefined;
```

Comment:
See the above about requiredState for the second tower.

Reply:
See #234